diff -up dragon-25.08.1/src/main.cpp.omv~ dragon-25.08.1/src/main.cpp
--- dragon-25.08.1/src/main.cpp.omv~	2025-09-11 19:14:49.766054863 +0200
+++ dragon-25.08.1/src/main.cpp	2025-09-11 19:15:57.925561615 +0200
@@ -24,7 +24,7 @@ int main(int argc, char **argv)
     if (Renderer::isAMD()) {
         // https://bugreports.qt.io/browse/QTBUG-138679
 #if QT_VERSION >= QT_VERSION_CHECK(6, 10, 0)
-#error "Reevaluate the use of QT_DISABLE_HW_TEXTURES_CONVERSION at a later point in time"
+#warning "Reevaluate the use of QT_DISABLE_HW_TEXTURES_CONVERSION at a later point in time"
 #endif
         qWarning() << "Detected AMD GPU, disabling HW Texture Conversion renderer as it is known to cause issues.";
         qputenv("QT_DISABLE_HW_TEXTURES_CONVERSION", "1");
